locale/en/download/package-manager.md

```diff
@@ -289,6 +290,28 @@ Or install a binary package (if available for your platform) using pkgin:
 pkgin -y install nodejs
 ```
 
+## nvm
+On Unix / OS X systems Node.js built from source can be installed using
```

Comment:
Could you please add a sentence or two about installing nvm in general and using it to install Node.js, not just how to add builds from source to it? Thx.

Otherwise LGTM.